.github/workflows/CI.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  push:
+    branches-ignore:
+      - 'staging.tmp'
+      - 'trying.tmp'
+
+name: CI
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [
+          1.37.0, # MSRV
+          stable,
+          beta,
+          nightly
```

Comment:
❓Do we need to test against all of them?

Reply:
We don't _need_ to but it is a good way of catching regressions early. From what I now, it is considered best practice in the Rust ecosystem because of the speed with which they release new compiler versions.